### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sibasankarnayak**
asdasd

### CODE REVIEW

**Comment by sweeneysiba** [comment]
> asdasd

asdasdasd test comment

**Comment by sweeneysiba** [comment]
> asdasd

hello

**Comment by sweeneysiba** [comment]
asdasd

**Review by sibasankarnayak — COMMENTED** [review]
asdasd

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 # Mattermost GitHub Plugin
 
 [![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-github/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-github)
-[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-github/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-github)
+[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-github
+
```
Comment (sibasankarnayak) [inline README.md:5]: dadsasd

**Comment by sibasankarnayak** [comment]
This is my first commit

This is the content of my 3rd line
#3 
This is the content of my 4th line, not 5th

**Pull request closed without merging**